ui/admin/app/components/workers/worker/actions/index.hbs

```diff
@@ -0,0 +1,20 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Rose::Dropdown
```

Comment:
Not a blocker- Can we replace this with HDS?

Reply:
I was thinking of making it an HDS dropdown but I don't think any of our other "manage" dropdown uses have been updated to HDS either. I opted to not change it so it doesn't stand out. Might be an easy change across the board though because I don't think we have that many "manage" dropdowns.

Reply:
I see that we are updating these in other places like multi-scope grants. I will update it here as well.